packages/compass-import-export/src/csv/csv-utils.spec.ts

```diff
@@ -877,3 +886,260 @@ describe('formatCSVHeaderName', function () {
     expect(formatCSVHeaderName(pathParts)).to.equal('foo[0].bar[1][2]');
   });
 });
+
+function compareTypes(
+  selectedType: CSVParsableFieldType,
+  types: (CSVParsableFieldType | 'undefined')[],
+  expected: (CSVParsableFieldType | 'undefined')[]
+) {
+  types.sort();
+  expected.sort();
+  try {
+    expect(types, selectedType).to.deep.equal(expected);
+  } catch (err) {
+    console.dir(types);
```

Comment:
This console.dir() leaves you with an array that's easy to copy/paste into code when we change the types. The diff shown by the error is a bit of a pain to work with alone in that context.